cpp/unreal_plugins/SimulationController/Source/SimulationController/SimulationController.cpp

```diff
@@ -89,8 +83,11 @@ void SimulationController::postWorldInitializationEventHandler(UWorld* world, co
     std::cout << "[SPEAR | SimulationController.cpp] SimulationController::postWorldInitializationEventHandler" << std::endl;
 
     ASSERT(world);
-
+#if WITH_EDITOR
```

Comment:
Note the white space. This would be more consistent with our existing code.

```
    #if WITH_EDITOR
        bool ready_to_open_level = world->IsGameWorld();
    #else
        bool ready_to_open_level = world->IsGameWorld() && GEngine->GetWorldContextFromWorld(world);
    #endif

    if (ready_to_open_level) {
        // ...
    }
```

Reply:
fixed

Reply:
Not yet fixed. Please indent the compile-time if statement as though it was a run-time if statement for consistency with our existing code (CameraSensor.cpp, CameraAgent.cpp, OpenBotAgent.cpp, etc.

Reply:
my bad. Misunderstood the format requirement.